src/webviews/webview-side/ipywidgets/common/manager.ts

```diff
@@ -188,10 +188,13 @@ export class WidgetManager implements IIPyWidgetManager, IMessageHandler {
         // eslint-disable-next-line @typescript-eslint/no-require-imports
         const jupyterLab = require('@jupyterlab/services') as typeof import('@jupyterlab/services'); // NOSONAR
 
-        if (!jupyterLab.KernelMessage.isDisplayDataMsg(payload)) {
+        if (
+            !jupyterLab.KernelMessage.isDisplayDataMsg(payload) &&
+            !jupyterLab.KernelMessage.isExecuteResultMsg(payload)
```

Comment:
Slight change in their widget protocol, now execute results contain the widget output for rendering